src/screens/Batches/CreateBatch/CreateBatch.tsx

```diff
@@ -6,15 +6,35 @@ import AntDesign from "react-native-vector-icons/AntDesign";
 import { TextInput } from "react-native-paper";
 import MyText from "../../../components/MyText";
 import { Colors } from "../../../constants/Colors";
-const CreateBatch = () => {
-  const [batchName, setBatchName] = React.useState("");
+import { useDispatch } from "react-redux";
+import { AppDispatch } from "../../../store";
+import { addBatch } from "../../../services/batches";
+
+const CreateBatch = ({ navigation }) => {
+  const goToBacthesScreen = () => navigation.navigate("Batches");
+  const [batchName, setBatchName] = React.useState(""); // useState
   const [batchDescription, setBatchDescription] = React.useState("")
```

Comment:
I would suggest to add some TS herer check the response use The AI and create interface and type the useState